src/complex/Utilities/TimeUtilities.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "complex/Utilities/StringUtilities.hpp"
+
+#include <fmt/core.h>
+#include <fmt/ranges.h>
+#include <string>
+
+namespace complex
+{
+inline std::string ConvertMillisToHrsMinSecs(unsigned long long int millis)
```

Comment:
Here I would make use of the std::chrono module to do the conversions for you with duration_casts. You could completely avoid stringUtilities.hpp. A good example of this would be here: https://stackoverflow.com/a/64169432/20736404